src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return rent * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
I think, that it will still work like this. Try it.
```suggestion
  if (days < 7) {
```

Reply:
You are totally right)